optuna/visualization/parallel_coordinate.py

```diff
@@ -110,6 +116,7 @@ def _get_parallel_coordinate_plot(study, params=None):
     traces = [
         go.Parcoords(
             dimensions=dims,
+            labelangle=labelangle,
```

Comment:
How about avoiding the overlapping with the title by placing the labels at the bottom? https://plot.ly/python/reference/#parcoords-labelside